kili/cli/project/member/list_.py

```diff
@@ -0,0 +1,76 @@
+"""CLI's project list subcommand"""
+
+from typing import Dict, List, Optional, cast
+import click
+
+import pandas as pd
+from tabulate import tabulate
+
+from kili.client import Kili
+from kili.cli.common_args import Options
+
+ROLE_ORDER = {
+    v: i for i, v in enumerate(["ADMIN", "TEAM_MANAGER", "REVIEWER", "LABELER"])
+}
+
+
+def role_order(col: pd.Series) -> pd.Series:
```

Comment:
can't you rather use a lambda function for the key is sort_values ?

Reply:
I am sure it is possible, but not that easy when you want to order by custom Role then alphabetically (if you have an idea, I am all hear)

Reply:
'''
users2 = users.sort_values(
    by=["ROLE", "lastname"],
    ascending=True,
    key=lambda column: column.map(ROLE_ORDER) if column.name =="ROLE" else column,
)
'''